src/app/views/sap/register.php

```diff
@@ -1,54 +1,107 @@
 <!DOCTYPE html>
-<?php
-?>
 <html lang="en">
 <head>
     <meta charset="utf-8">
     <title>Student Ambassador Program · Felicity ʼ16</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="<?= base_url() ?>static/styles/vendor/thoda.min.css">
-    <link rel="stylesheet" href="<?= base_url() ?>static/styles/sap_register.css">
+    <link rel="stylesheet" href="https://felicity.iiit.ac.in/css/vendor/thoda.min.css">
```

Comment:
Yes, this is very bad. But the deploy was so painful, I ended up doing this. Needs fixing.